packages/pixeloven-webpack/config/src/index.ts

```diff
@@ -179,6 +243,24 @@ function getConfig(options: Options) {
             rules: [
                 {
                     oneOf: [
+                        /**
+                         * Sets express 4.x view to null. We use our own custom react renderer and have no use for this feature.
+                         * Ideally express would make this modular but with this we can optimize express for SSR.
+                         *
+                         * @note Side effect is none of the features associated with express template rendering will work.
+                         * https://expressjs.com/en/api.html#res.render
+                         */
+                        {
+                            test: /express\/lib\/view.js$/,
```

Comment:
This eliminates the critical dependency warning by basically replacing this module with null.
```
var View = require('./view');
// View === null
```
which can be found in ./node_modules/express/lib/application.js

Since we are rendering react on the server with our own custom renderer we have no need for expresses build in view/tempting system. Not only does this cause us issues when trying to bundle express into a single deliverable but it also becomes an optimization for us.